src/components/Picklist/__test__/picklist.spec.js

```diff
@@ -24,43 +24,6 @@ describe('<Picklist />', () => {
         );
         expect(component.find('input').prop('value')).toBe('Option 1');
     });
-    it('should have the right number of childs registered when mounted', () => {
-        const optionRegisterChildFn = jest.spyOn(Picklist.prototype, 'registerChild');
-        const component = mount(
-            <Picklist label="Picklist">
-                <PicklistOption label="Option 1" name="option1" />
-                <PicklistOption disabled label="Option 2" name="option2" />
-                <PicklistOption variant="header" label="Header 3" />
-                <PicklistOption label="Option 3" name="option3" />
-                <PicklistOption label="Option 4" name="option4" />
-            </Picklist>,
-        );
-        jest.runAllTimers();
-        expect(component.instance().activeChildren.length).toBe(3);
-        expect(optionRegisterChildFn).toHaveBeenCalledTimes(3);
```

Comment:
no, what I mean is that this last expect where you test that registerChild is called is what is already tested but the length of the activeChildren is fine and is necessary